ledger-tool/src/main.rs

```diff
@@ -172,6 +172,7 @@ impl Default for GraphVoteAccountMode {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
This one looks like it can be addressed by removing the inner `String` and just doing:
```rust
struct GraphVoteAccountModeError;
```

Reply:
okay! [16dc55e](https://github.com/anza-xyz/agave/pull/1246/commits/16dc55e712899565dbd4780448e1cdcad2a07f44)